server/models/Profile.js

```diff
@@ -0,0 +1,50 @@
+const { Schema } = require('mongoose');
+
+const profileSchema = new mongoose.Schema({
+
+  user: {type: Schema.Types.ObjectId,
```

Comment:
```suggestion
  user: {
  type: Schema.Types.ObjectId,
  required: true
```

This is supposed to be a required field :)